dashboards/ruby_vm/main.json

```diff
@@ -0,0 +1,164 @@
+{
+  "metric_keys": [
+    "total_allocated_objects"
+  ],
+  "dashboard": {
+    "title": "MRI",
```

Comment:
Isn't this title a bit too generic? What if we want to add another ruby related dashboard in the future?

Reply:
I think it makes sense. If we add more MRI metrics later we could add that here.